docs/guides/cloud/introduction.mdx

```diff
@@ -128,12 +131,14 @@ directly from specific test failures.
 />
 
 Lastly you can use our flexible adminstrative functions to configure Cypress
-Cloud however you want, grouping [projects](/guides/cloud/projects) into
-multiple [organizations](/guides/cloud/organizations) if you have a lot,
-checking [usage](/guides/cloud/organizations#Billing-Usage), and administering
-[users and permissions](/guides/cloud/users#User-roles). We also provide
-[SSO integration](/guides/cloud/organizations#Enterprise-SSO) for teams on our
-[Enterprise plan](https://www.cypress.io/pricing/).
+Cloud however you want, grouping
+[projects](/guides/cloud/account-management/projects) into multiple
+[organizations](/guides/cloud/account-management/organizations) if you have a
+lot, checking [usage](/guides/cloud/account-management/billing-and-usage), and
+administering
+[users and permissions](/guides/cloud/account-management/users#User-roles). We
+also provide [SSO integration](/guides/cloud/account-management/enterprise-sso)
+for teams on our [Enterprise plan](https://www.cypress.io/pricing/).
```

Comment:
There are so many links in this paragraph. Do people actually click through these?

Reply:
This content will change in the updated version of this page https://github.com/cypress-io/cypress-documentation/pull/5121